File: rest/voice/outbound-calls/example-2/example-2.6.x.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest import Client

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token = "your_auth_token"

client = Client(account_sid, auth_token)

call = client.api.v2010.account.calls.create(url="http://demo.twilio.com/docs/voice.xml",
                                   to="client:tommy",
                                   from_="+15017250604")

print(call.sid)

File: rest/making-calls-sip/example-2/example-2.6.x.rb
# Get twilio-ruby from twilio.com/docs/ruby/install
require 'twilio-ruby'

# Get your Account SID and Auth Token from twilio.com/console
# To set up environmental variables, see http://twil.io/secure
account_sid = ENV['TWILIO_ACCOUNT_SID']
auth_token = ENV['TWILIO_AUTH_TOKEN']

# Initialize Twilio Client
@client = Twilio::REST::Client.new(account_sid, auth_token)

# Get the accounts with the given sid
@accounts = @client.api.v2010.accounts(account_sid)

# Create a call sip w/ authentication
@call = @client.calls.create(
  url: 'http://www.example.com/sipdial.xml',
  to: 'sip:kate@example.com',
  from: 'Jack',
  sip_auth_password: 'secret',
  sip_auth_username: 'jack'
)

# Print call recipient
puts @call.to
